examples/echo_server.html

```diff
@@ -6,7 +6,7 @@
     <script>
         let NUM_PACKETS = 20;
         let SEND_INTERVAL = 500;
-        let ADDRESS = "/new_rtc_session";
+        let ADDRESS = "//127.0.0.1:42425/new_rtc_session";
```

Comment:
Is there a reason for adding the specifc address here, this would make the example fail if the command was not exactly what was in the readme (otherwise it should continue to work fine?)